src/Tizen.NUI.Components/Controls/ItemsView/ItemsView.cs

```diff
@@ -0,0 +1,454 @@
+        {
+            get
+            {
+                return _itemsTemplate;
+            }
+            set
+            {
+                _itemsTemplate = value;
+            }
+        }
+
+        private ItemsLayouter _itemsLayouter = null;
+        /// <summary>
+        /// Items Layouter.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ItemsLayouter ItemsLayouter
+        {
+            get
+            {
+                return _itemsLayouter;
+            }
+            set
+            {
+                _itemsLayouter = value;
+            }
+        }
+
+        /// <summary>
+        /// RecycleCache of ViewItem.
+        /// </summary>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        protected List<ViewItem> RecycleCache = new List<ViewItem>();
+            
+        /// <summary>
+        /// Max size of RecycleCache. Default is 50.
+        /// </summary>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        protected int CacheMax = 50;
+
+        /// <inheritdoc/>
+         public override void OnRelayout(Vector2 size, RelayoutContainer container)
+        {
+            //Console.WriteLine("LSH :: On ReLayout [{0} {0}]", size.X, size.Y);
+            base.OnRelayout(size, container);
+            if (ItemsLayouter != null && ItemsSource != null && ItemsTemplate != null) 
+            {
+                ItemsLayouter.Initialize(this);
+                ItemsLayouter.RequestLayout(ScrollingDirection == Direction.Horizontal ? ContentContainer.CurrentPosition.X : ContentContainer.CurrentPosition.Y, true);
+            }
+        }
+
+        private void OnScrolling(object source, ScrollEventArgs args)
+        {
+            if (ItemsLayouter != null && ItemsSource != null && ItemsTemplate != null)
+            {
+            //Console.WriteLine("LSH :: On Scrolling! {0} => {1}", ScrollPosition.Y, args.Position.Y);
+                ItemsLayouter.RequestLayout(ScrollingDirection == Direction.Horizontal ? args.Position.X : args.Position.Y);
+            }
+        }        
+
+        /// <summary>
+        /// Push the item into the recycle cache. this item will be reused in view update.
+        /// </summary>
+        /// <param name="item"> Target item to push into recycle cache. </param>
+        /// <since_tizen> 8 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected virtual bool PushRecycleCache(ViewItem item)
+        {
+            if (RecycleCache.Count >= CacheMax) return false;
+            if (item.Template == null) return false;
```

Comment:
item의 null 확인 코드 넣어주세요.
```suggestion
            if (item?.Template == null) return false;
```
OR
```cs
            if (item == null)
                throw new ArgumentNullException(nameof(item));
```
ACR과정에서 testcase 작성시에 negative case로 exception을 요구하기 때문에 `ArgumentNullException` 추가하는 것을 추천드립니다.